src/Acme.UI/Controllers/Business/AthletesController.cs

```diff
@@ -1,31 +1,69 @@
-﻿using Examine;
-using Lucene.Net.Search;
+﻿using System.Collections.Generic;
 using System.Linq;
 using System.Web.Mvc;
+using Umbraco.Web;
 using Umbraco.Web.Mvc;
-
+using Examine;
+using Umbraco.Examine;
 
 namespace Acme.UI.Controllers.Business
 {
     public class AthletesController : SurfaceController
     {
+        public ActionResult GetAthlete(Umbraco.Web.PublishedModels.Athlete athlete)
+        {
+            return View("~/Views/Athletes/Athlete.cshtml", athlete);
+        }
+
+
+        public ActionResult AthleteDetails(int athleteId)
+        {
+            var athlete = Umbraco.Content(athleteId);
+            return View("~/Views/Athletes/Athlete.cshtml", athlete);
+        }
+
         // GET: Athletes
         public ActionResult List(int? maxAthletesToDisplay)
         {
-            if (ExamineManager.Instance.TryGetIndex("ExternalIndex", out var index))
+            var logginPhysioNodeId = GetLoggedInPhysioNodeId();
+            var athletes = new List<Umbraco.Web.PublishedModels.Athlete>();
```

Comment:
add that namespace into a using statement please and remove the fully qualified names in this file

Reply:
Done